srcpkgs/gscreenshot/template

```diff
@@ -1,15 +1,18 @@
 # Template file for 'gscreenshot'
 pkgname=gscreenshot
-version=2.17.1
-revision=2
+version=3.3.0
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools gettext"
+hostmakedepends="python3-setuptools gettext go-md2man"
 depends="gtk+3 python3-setuptools python3-Pillow python3-gobject scrot"
-short_desc="GUI front-end for the scrot program"
+checkdepends="${depends} python3-mock python3-pytest"
+short_desc="GUI for multiple screenshot backends including scrot, PIL, and grim"
 maintainer="Rui Flora <rui.flora@gmail.com>"
-license="GPL-2.0-or-later"
+license="GPL-2.0-only"
```

Comment:
could you ask about this upstream?

Reply:
https://github.com/thenaterhood/gscreenshot/issues/166